components/greaselion/browser/greaselion_service_impl.cc

```diff
@@ -27,8 +27,8 @@
 #include "base/task_runner_util.h"
 #include "base/values.h"
 #include "base/version.h"
-#include "brave/components/brave_component_updater/browser/features.h"
-#include "brave/components/brave_component_updater/browser/switches.h"
+#include "brave/common/brave_features.h"
```

Comment:
this change seems backwards, why would we move something component updater specific from components -> common? That creates a layering violation where none existed before

Reply:
resolved!